sim/dex-moves.ts

```diff
@@ -240,7 +240,7 @@ export interface MoveData extends EffectData, MoveEventMethods, HitEffect {
 	baseMove?: string;
 }
 
-export type ModdedMoveData = MoveData | Partial<Omit<MoveData, 'name'>> & {inherit: true};
+export type ModdedMoveData = MoveData | Partial<Omit<MoveData, 'name'>> & {inherit: true, gen?: number};
```

Comment:
@Zarel is this unnecessary as well?

Reply:
@Zarel bump ;w;

Reply:
no

Reply:
It should be imo. It not being unnecessary seems to be a bug?

Reply:
its not a bug, currently gen is decided off of a move's index number, which these custom moves dont have

Reply:
it's just outdated code